tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "target": "es6",
+    "lib": ["es2017"],
```

Comment:
why including this?

Reply:
Because Array.prototype.includes is a feature of ES7 ,   
typescript compiles with an error, if es2017 is not introduced